docs/src/webknossos-py/examples/download_image_data.md

```diff
@@ -0,0 +1,9 @@
+# Upload Image Data
+
+Downloads part of a [webknossos dataset](https://webknossos.org/datasets/scalable_minds/l4dense_motta_et_al_demo/view?utm_source=site#2867,4361,1770,0,1.21) from a local webknossos instance using [`webknossos_context`](../../api/webknossos/client/context.md#webknossos_context).
```

Comment:
Should we set the `utm_source=site` in the docs now or is this copied from somewhere else? cc @normanrz 

```suggestion
Downloads part of a [webknossos dataset](https://webknossos.org/datasets/scalable_minds/l4dense_motta_et_al_demo/view) from a local webknossos instance using [`webknossos_context`](../../api/webknossos/client/context.md#webknossos_context).
```

Reply:
No, utm_source doesn't need to be part of links in the docs.